### ISSUE TITLE
Comments in generated code

### ISSUE THREAD

**Issue by mehmetsalgar** [pascaldekloe/colfer#39]
Hi,

colfer place the name of schema file to the comment it generates via Maven Plugin, I am using  a Windows machine and colfer does not escape the "\" in file path and java compiler then complains about 'illegal escape character', is there a way to disable this comment or let it the colfer  escape it before putting in the comment?

Thx

**Comment by pascaldekloe** [comment]
Understood. Can you share the generated comment line which causes the Java compiler to fail?
You'll have a solution by tomorrow.

**Comment by mehmetsalgar** [comment]
sure

```
@javax.annotation.Generated(value="colf(1)", comments="Colfer from schema file D:\workspace\sample_colfr\org\test\colfer\vo\ProductContainer.java")
public class ProductContainer implements Serializable {

}
```

**Comment by pascaldekloe** [comment]
Right, you mean those annotations. Yes they must be escaped of course. No problem.

**Comment by pascaldekloe** [comment]
Does this fix work for you @mehmetsalgar? Can you use a snapshot or shall I build a release (including Maven pugin)?

**Comment by ginopablo** [comment]
when will this fix be released for maven-plugin? thanks in advance.

**Comment by pascaldekloe** [comment]
Most likely in a few months from now. If that's too late for you @ginpablo then I will make a release now. Takes about 40 minutes or so.

**Comment by ginopablo** [comment]
Thank for your reply. Yes that's too late for me. if i can get a release, i'm keeping colfer in our application.

**Comment by pascaldekloe** [comment]
Done @ginopablo. ✔️  The Maven plugin version 1.11.1 should be in central shortly.

**Comment by ginopablo** [comment]
It works of course! Great support, thanks @pascaldekloe
